Gems/EMotionFX/Code/Source/Editor/Plugins/Ragdoll/RagdollManipulators.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+#include <AzCore/Math/Transform.h>
+#include <QWidget>
+
+namespace Physics
+{
+    class CharacterColliderNodeConfiguration;
+} // namespace Physics
+
+namespace EMotionFX
+{
+    class Actor;
+    class Node;
+    class ObjectEditor;
+    class ColliderContainerWidget;
+
+    struct RagdollManipulatorData
+    {
+        AZ::Transform m_nodeWorldTransform = AZ::Transform::CreateIdentity();
+        Physics::CharacterColliderNodeConfiguration* m_colliderNodeConfiguration = nullptr;
+        Actor* m_actor = nullptr;
+        Node* m_node = nullptr;
+        ColliderContainerWidget* m_collidersWidget = nullptr;
+        bool m_valid = false;
+    };
+
+    //! Base class for various manipulator modes, e.g. collider translation, collider orientation, etc.
+    class RagdollManipulatorsBase
```

Comment:
Replace Ragdoll with Collider? As this is for more than just the ragdoll colliders

Reply:
Yeah, was thinking about that myself. Renamed a whole bunch of things from Ragdoll... to PhysicsSetup...